src/main/java/gov/acwi/wqp/etl/methWithCit/TaskMethWithCitMeth.java

```diff
@@ -0,0 +1,56 @@
+package gov.acwi.wqp.etl.methWithCit;
```

Comment:
I implemented this with a tasklet because the standard reader failed mysteriously when trying to perform the Min aggregation